### ISSUE TITLE
Impossible to preview the content in the video section

### ISSUE THREAD

**Issue by eskander1998** [layerhub-io/react-design-editor#14]
![image](https://user-images.githubusercontent.com/47850204/185584486-a75a255e-b653-4079-bd1e-5e52378ec52c.png)

Generating prevue doesn't work on video section , it keeps loading with no result.

**Comment by chloe-bm** [comment]
@eskander1998 can you share exported JSON ?

**Comment by prince214** [comment]
@chloe-bm yes, i am facing the same issue. in console its saying video too large. i uploaded only 2mb video still giving me error. I am attaching the json in zip format.
[template.zip](https://github.com/layerhub-io/react-design-editor/files/10491356/template.zip)

![image](https://user-images.githubusercontent.com/33788664/214332295-76e6d30d-e5ce-4314-8e9f-7315d3c83be4.png)

**Comment by DongJu-Na** [comment]
+1
